src/backer/cardaning.py

```diff
@@ -143,18 +138,15 @@ def confirmTrans(self):
                     continue
 
                 item = json.loads(item)
-                (blockSlot, blockHeight) = (item["block_slot"], int(item["block_height"])) if 'block_slot' in item.keys() else (0, 0)
-                publishTime = datetime.fromisoformat(item["publish_time"])
-                onChainTime = help.toIso8601(self.slotToTime(blockSlot))
-                onChainTime = datetime.fromisoformat(onChainTime).replace(tzinfo=timezone.utc)
-                confirmTime = datetime.fromisoformat(help.toIso8601())
+                transTip = int(item["tip"])
+                blockHeight = int(item["block_height"]) if 'block_height' in item.keys() else 0
 
                 # Check for confirmation
                 if self.tipHeight > 0 and blockHeight > 0 and self.tipHeight - blockHeight >= TRANSACTION_SECURITY_DEPTH:
                     self.keldbConfirming.rem(keys)
                     continue
 
-                if (confirmTime - publishTime).total_seconds() > TRANSACTION_CONFIRMATION_TIMEOUT:
+                if self.tipHeight - transTip > TRANSACTION_TIMEOUT_DEPTH:
```

Comment:
Should we not be comparing the current tip and the `blockHeight` of the tx? What's the `transTip` - is the value the same when submitting a tx?

Reply:
>Should we not be comparing the current tip and the blockHeight of the tx?
We don't know the blockHeight when tx has not appeared in the crawler yet
>is the value the same when submitting a tx?
Yes, It is

Reply:
sorry yeah makes sense